### PR TITLE
Minor type fixes

### PR DESCRIPTION
**Pull request by mvdbeek**
I got a little side-tracked and attempted to type things in `model/__init__.py` and found these issues.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: lib/galaxy/celery/tasks.py [hunk 1]
```diff
@@ -83,15 +83,15 @@ def export_history(
...
+        store_directory: str,
+        history_id: int,
+        job_id: int,
         include_hidden=False,
         include_deleted=False):
     history = sa_session.query(model.History).get(history_id)
     with model.store.DirectoryModelExportStore(store_directory, app=app, export_files="symlink") as export_store:
         export_store.export_history(history, include_hidden=include_hidden, include_deleted=include_deleted)
-    job = sa_session.query(model.Job).get(job_id)
+    job = sa_session.query(model.Job).filter_by(id=job_id).one()
```
Comment (jdavcs) [inline lib/galaxy/celery/tasks.py:94]: This is a minor thing, but I think that would then force a select even if the object is in the session. If we need to ensure an error is raised if no job is found, maybe raise it explicitly if the `get` returns `None`?

**Review by jdavcs — APPROVED** [review]
..And a very enthusiastic +1 for all the `trans.app.model` >> `galaxy.model`!

File: lib/galaxy/celery/tasks.py (hunk 1 above)
Comment (mvdbeek) [reply lib/galaxy/celery/tasks.py:94]: Hmm, that is an interesting point: We should probably ensure tasks receive a fresh session, but I think the scoping mechanism we have now will just ensure that the process, thread or request get their own session.

Comment (jdavcs) [reply lib/galaxy/celery/tasks.py:94]: Yes, that's my understanding of how scoped_session works too. I think if we needed a separate session, we could use `session_factory` stored in the `scoped_session` to create one ([docs](https://docs.sqlalchemy.org/en/20/orm/contextual.html#sqlalchemy.orm.scoping.scoped_session.session_factory))

**Comment by mvdbeek** [comment]
Thanks @ic4f, let's follow up on the session management later.

**Pull request merged**